import/_all.scss

```diff
@@ -4,9 +4,11 @@
 @import "../src/bootstrap-customisations/accordion";
 @import "../src/bootstrap-customisations/alerts";
 @import "../src/bootstrap-customisations/buttons";
+@import "../src/bootstrap-customisations/breadcrumb";
```

Comment:
breadcrumb?

Reply:
Since we haven't made a new release, the breadcrumb from previous sprint is fetched from the demo-repo.

Reply:
Should be ok now